readthedocs/notifications/messages.py

```diff
@@ -355,6 +344,7 @@ def get_rendered_body(self):
             textwrap.dedent(
                 f"""
              PDF file was not generated/found in "{BUILD_COMMANDS_OUTPUT_PATH_HTML}/pdf" output directory.
+             Make sure your are saving the the PDF file in the correct directory.
```

Comment:
Typo at `the the` and this could be reduced to remove indirect subject. It might be redundant to the first line, but I think that is mostly fine too.

```suggestion
             Make sure the PDF file is saved in this directory.
```